Core/src/org/sleuthkit/autopsy/centralrepository/datamodel/CentralRepository.java

```diff
@@ -828,4 +874,8 @@ CorrelationAttributeInstance getCorrelationAttributeInstance(CorrelationAttribut
     CentralRepoAccount getOrCreateAccount(CentralRepoAccount.CentralRepoAccountType crAccountType, String accountUniqueID) throws CentralRepoException;
     
              
+    /**
+     * Clears all caches.
+     */
+    void clearCaches();
```

Comment:
As it happens, although the setup() method in the test that gets called between EVERY test method wipes out the underlying database before the start of each test, the CentralRepository object, as returned by CentralRepository.getInstance(), is only created once.  With each invocation of the setup method, the caches held by the CentralRepository instance go out of sync and inconsistent with the database, causing errors due to database integrity.

Since we've taken out caching of Examiners, this *may* not be necessary at the moment though.